### PR TITLE
First project

### PR DESCRIPTION
**Pull request by shakerAbuDrais**
This is my first pull request where I created 3 files (HTML, CSS, Test) the last file was ignored and that was my first project

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,17 @@
...
+all you need to do is to just open the HTML file, copy it to your local machine and test it out, this is a simple project. 
+
+## Authors 
+Github: @shakerAbuDrais
+
+## 🤝 Contributing
+## Show your support
+Give a ⭐️ if you like this project!
+
+# 📝 License
```
Comment (victorokoroji) [inline README.md]: You have done well updating your readme 😄, but I think you should improve on your readme by following the [template](https://github.com/microverseinc/readme-template/blob/master/README.md) provided by Microverse. Please ensure your readme looks like the template provided 👍🏼 .

File: .gitignore [hunk 2]
```diff
@@ -0,0 +1 @@
+test.md
```
Comment (victorokoroji) [inline .gitignore:1]: - [ ] Kindly remember to add node_modules to gitignore which will be generated while setting up linters in your project 😄.

File: README.md (hunk 1 above)
Comment (victorokoroji) [inline README.md:17]: - [ ] You have done well updating your readme 😄, but I think you should improve on your readme by following the [template](https://github.com/microverseinc/readme-template/blob/master/README.md) provided by Microverse. Please ensure your readme looks like the template provided 👍🏼 .

**Review by victorokoroji — CHANGES_REQUESTED** [review]
Hi @shakerAbuDrais, 

You have done a great job so far! 💪 👏
You still need to work on a few issues before moving on to the next project, but you're almost there!

## Top highlight 🎯

- The Pull Request is following the GitHub Flow ✅ 
- The PR title and description are professional and descriptive ✅  

## Required Changes ♻️

- [ ] I think it will be better if you write more descriptive commit messages in imperative tense with the first letter of your commit message capitalized 😄 . This will make your repo look more professional.
Instead of writing: **HTML file created with H1**
You can write: **Create an HTML file with H1**

- [ ] You did well adding the linter workflow 👍🏼 . Kindly follow this [link](https://github.com/microverseinc/linters-config/tree/master/html-css) to set up linters correctly. Currently, your linters are lacking the stylelint and the web hint config files 👍🏼 .

_Check the comments under the review._


## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:

N/A

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by shakerAbuDrais** [comment]
hi @vickymarz 
thank you so much for the insights, in regards to the linters, they took me more than 1 hour to fix them, there were some issues with CSS, and CSS linters asked me to fix these issues, in regard to the comments issues when i use this phrase "Create an HTML file with H1" it sounds like am asking someone to do it, i used present past to describe what I have done in the file already

**Comment by shakerAbuDrais** [comment]
@vickymarz can you please share a link with me on how can i remove the commits and rewrite them again? or should i redo the whole project?

File: README.md (hunk 1 above)
Comment (shakerAbuDrais) [reply README.md:17]: @vickymarz  just replaying here to make sure you see my comments below

**Comment by victorokoroji** [comment]
@shakerAbuDrais for the sake of your upcoming commits, the best way to write a commit is to write in imperative tense as if you are instructimg someone to to do it, and it should be descriptive.

**Comment by shakerAbuDrais** [comment]
> @shakerAbuDrais for the sake of your upcoming commits, the best way to write a commit is to write in imperative tense as if you are instructimg someone to to do it, and it should be descriptive.

@vickymarz noted mate, I will stick to what you said, I Have been looking online how to amend my current commits and found out that I can do rebase but I could not do it right, so till the moment I have no idea how to change the commits, sorry if am taking of your time but is there away to change the commits or should I redo the project?

**Comment by victorokoroji** [comment]
@shakerAbuDrais you can't change all your commits. You can only update your last commit.
Don't bother trying to change your commits . Just introduce improvements in your next commits 😄

**Comment by shakerAbuDrais** [comment]
> @shakerAbuDrais you can't change all your commits. You can only update your last commit. Don't bother trying to change your commits . Just introduce improvements in your next commits 😄

@vickymarz for the linters I have checked them again against the link you shared and it turned out that all of them are on, I used the linters on github and I opened the details for each section.

**Comment by shakerAbuDrais** [comment]
image
as you can see I have 4 successful checks!

**Comment by shakerAbuDrais** [comment]
> image as you can see I have 4 successful checks!

@vickymarz

**Comment by victorokoroji** [comment]
@shakerAbuDrais you haven't added the .stylintrc.json and .hintrc config files for CSS and html. Once you add them, then you are good to go. Check the linter link I sent and follow it step by step

**Review by ShahierNashaat — APPROVED** [review]
Hi @shakerAbuDrais, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## To Highlight 💯 

- Correct use of GitFlow ✔️
- Descriptive commit messages ✔️
- Professional looking pull request ✔️

# Approved :ok_hand: :ok_hand: :ok_hand:

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**